### ISSUE TITLE
PR #1451 has caused some cime_developer tests to fail during phase MODEL_BUILD

### ISSUE THREAD

**Issue by dqwu** [ESMCI/cime#1456]
The build errors can be reproduced on anlworkstation, blues and edison by running a single cime_developer test (use GNU compiler) :
./create_test NCK_Ld3.f45_g37_rx1.A --machine=**** --compiler=gnu --test-root=****

ERROR: ocn.buildlib failed (NOT reproducible with intel compiler)
```
src/components/data_comps/docn/docn_comp_mod.F90:305:34:

           read(ocn_mode(10:10),'(i)') aquap_option
                                  1
Error: Nonnegative width required in format string at (1)
src/components/data_comps/docn/docn_comp_mod.F90:307:34:

           read(ocn_mode(10:11),'(i)') aquap_option
                                  1
Error: Nonnegative width required in format string at (1)
```

**Comment by jedwards4b** [comment]
This is fixed in PR #1453

**Comment by dqwu** [comment]
PR #1453 works (it is not merged to master so far).
It seems that gfortran and nag Fortran compiler need a width, but intel Fortran compiler does not.
